File: tests/fixtures/javascript/clean.js
function add(a, b) {
    return a + b;
}

function greet(name) {
    return `Hello, ${name}`;
}

class Calculator {
    constructor(value = 0) {
        this.value = value;
    }

    add(n) {
        this.value += n;
        return this;
    }

    subtract(n) {
        this.value -= n;
        return this;
    }

    result() {
        return this.value;
    }
}

File: tests/fixtures/rust/clean.rs
fn add(a: i32, b: i32) -> i32 {
    a + b
}

fn greet(name: &str) -> String {
    format!("Hello, {}", name)
}

struct Calculator {
    value: i32,
}

impl Calculator {
    fn new(value: i32) -> Self {
        Self { value }
    }

    fn add(&mut self, n: i32) -> &mut Self {
        self.value += n;
        self
    }

    fn subtract(&mut self, n: i32) -> &mut Self {
        self.value -= n;
        self
    }

    fn result(&self) -> i32 {
        self.value
    }
}
